### PR TITLE
Change all occurrences of master (request) to main

### PR DESCRIPTION
**Pull request by aschempp**
Related to https://github.com/contao/contao/pull/3436

### CODE REVIEW

**Review by fritzmg — DISMISSED** [review]

**Review by Toflar — DISMISSED** [review]

File: core-bundle/src/Controller/InitializeController.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function __invoke(): InitializeControllerResponse
     {
         trigger_deprecation('contao/core-bundle', '4.0', 'Using custom entry points has been deprecated and will no longer work in Contao 5.0.');
 
-        $masterRequest = $this->requestStack->getMasterRequest();
+        $masterRequest = $this->requestStack->getMainRequest();
```
Comment (bytehead) [inline core-bundle/src/Controller/InitializeController.php]: +1 to replace all occurences of `master*`  with `main*`.

**Review by bytehead — CHANGES_REQUESTED** [review]

File: core-bundle/src/Controller/InitializeController.php (hunk 1 above)
Comment (bytehead) [inline core-bundle/src/Controller/InitializeController.php]:
```suggestion
        $mainRequest = $this->requestStack->getMainRequest();
```

File: core-bundle/src/Controller/InitializeController.php [hunk 2]
```diff
@@ -83,7 +83,7 @@ public function __invoke(): InitializeControllerResponse
...
 
         if (null === $masterRequest) {
             throw new \RuntimeException('The request stack did not contain a master request.');
```
Comment (bytehead) [inline core-bundle/src/Controller/InitializeController.php]:
```suggestion
            throw new \RuntimeException('The request stack did not contain a main request.');
```

Comment (aschempp) [reply core-bundle/src/Controller/InitializeController.php]: Fixed in 46c12f9d122c998a7f7e24d7e0e39ad8c0aa693f

File: core-bundle/src/Controller/InitializeController.php (hunk 1 above)
Comment (aschempp) [reply core-bundle/src/Controller/InitializeController.php]: Fixed in 46c12f9d122c998a7f7e24d7e0e39ad8c0aa693f

Comment (aschempp) [reply core-bundle/src/Controller/InitializeController.php]: Fixed in 46c12f9d122c998a7f7e24d7e0e39ad8c0aa693f

**Review by bytehead — DISMISSED** [review]
🥳

**Review by bytehead — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Pull request merged**